gistBFO.ttl

```diff
@@ -10,13 +10,13 @@
 @prefix xml: <http://www.w3.org/XML/1998/namespace> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-<https://ontologies.semanticarts.com/gistBFO>
+<https://w3id.org/semanticarts/ontology/gistBFO>
 	a owl:Ontology ;
 	owl:imports
 		<http://purl.obolibrary.org/obo/bfo/2020/bfo-core.ttl> ,
 		<https://w3id.org/semanticarts/ontology/gistCore13.0.0>
 		;
-	owl:versionIRI <https://ontologies.semanticarts.com/gistBFO.0.1.0> ;
+	owl:versionIRI <https://w3id.org/semanticarts/ontology/gistBFO.0.1.0> ;
```

Comment:
With gist and client ontologies, we usually don't have a leading `.`. With 'gistBFO' I admit it's a little easier to read with the `.`. However, might be best to omit for the sake of consistency?
```suggestion
	owl:versionIRI <https://w3id.org/semanticarts/ontology/gistBFO0.1.0> ;
```